### PR TITLE
correct param, var and return annotation values

### PR DESCRIPTION
**Pull request by TomasVotruba**
This PR correct malformed annotations:

- switched `$type name`
- typos
- $this to `self` that is recognized type by IDE
- remove unused `$value` in `@var` and `@return` annotations

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=19325) <br/>All committers have signed the CLA.

**Comment by magento-engcom-team** [comment]
Hi @TomasVotruba. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by sivaschenko** [comment]
Code review passed, waiting for the builds to complete

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-3525](https://jira.corp.magento.com/browse/ENGCOM-3525) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
@TomasVotruba thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

File: lib/internal/Magento/Framework/Message/Manager.php
Comment (TomasVotruba) [inline lib/internal/Magento/Framework/Message/Manager.php]: Missed case, I'll fix this first

**Comment by TomasVotruba** [comment]
I've fixed one false positive. It's ready now

@sivaschenko Thanks for very fast review!

**Comment by TomasVotruba** [comment]
Failing Travis is not related to this PR 

Some *Timeout* - https://travis-ci.org/magento/magento2/jobs/458340630#L2557

**Comment by TomasVotruba** [comment]
What needs to be done to merge this?

**Comment by sivaschenko** [comment]
@TomasVotruba that pull request is currently in progress of delivery to mainline

**Comment by TomasVotruba** [comment]
Thanks, I didn't know that

**Comment by magento-engcom-team** [comment]
Hi @TomasVotruba. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.1. 
Please check the release notes for final confirmation.

**Comment by TomasVotruba** [comment]
Glad to help!

Btw, it always takes ~10 days to merge such simple changes?

I want to send more similar PRs. But I'm afraid there will be many new code in the 10 days period and it will get stuck in rebase-resolve conflicts infinite loop.

**Comment by sivaschenko** [comment]
@TomasVotruba I don't think it usually the case, there is a long internal queue of pull requests formed recently. Sorry for the delay.

**Comment by TomasVotruba** [comment]
@sivaschenko I see. Is there some description of this flow?

Usually is 1-2 accepted reviews by maintainers + 3rd one marges.

**Comment by sivaschenko** [comment]
@TomasVotruba after the pull request is approved on public GitHub it is moved to an internal repository for further QA and passing through internal CI test builds (it's only a small part of tests running on each pull request publicly).

@sidolov do we have any documentation on internal PR processing?

**Comment by sidolov** [comment]
Hi @sivaschenko , we haven't such info right now

**Pull request merged**
